### PR TITLE
Update character.md

### PR DESCRIPTION
**Pull request by mhalsubhi**
Add limit docs 

#50 #75 #74

### CODE REVIEW

File: frontend/docs/character.md [hunk 1]
```diff
@@ -118,4 +118,78 @@ https://finalspaceapi.com/api/v0/character/?sort=desc
 ]
 ```
 
+### Limit numbers of returend characters
```
Comment (lelouchB) [inline frontend/docs/character.md:121]: Spelling error returned
You can make it `Limit the response`

**Comment by lelouchB** [comment]
@mhalsubhi 
Also add this to `episode` docs
Add this example too

```
 https://finalspaceapi.com/api/v0/character/?limit=3$sort=desc
```

File: frontend/docs/character.md (hunk 1 above)
Comment (mhalsubhi) [reply frontend/docs/character.md:121]: ok 👍🏼

**Comment by mhalsubhi** [comment]
> @mhalsubhi
> Also add this to `episode` docs
> Add this example too
> 
> ```
>  https://finalspaceapi.com/api/v0/character/?limit=3$sort=desc
> ```

ok I will 👍🏼

**Review by lelouchB — APPROVED** [review]

**Pull request merged**